recipes/r-minems2/meta.yaml

```diff
@@ -18,19 +18,19 @@ requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - r-base
-    - r-rcpp
-    - bioconductor-msnbase
-    - r-stringr
-    - r-igraph
-    - r-matrix
+    - r-base>=3.5.1
```

Comment:
We only build for a single version of R at a time, since that's what all of the bioconductor packages need. So can you remove `>=3.5.1`?